RELEASE.md

```diff
@@ -0,0 +1,101 @@
+# Releases
+
+## Prerequisites - what you need to release WooCommerce Blocks
+- Install & set up [GitHub hub](https://hub.github.com) tools.
+- Configure a GitHub token for release scripts to use.
+  - https://github.com/settings/tokens
+  - Select the following permissions: 
+    - `admin:org`
+    - `public_repo`
+    - `read:user`
+  - Ensure your token is available to scripts, e.g. `export GH_API_TOKEN={YOUR-TOKEN}` or similar.
+- Get WPORG plugin svn commit access - ask a team member.
+
```

Comment:
>  I realize the use of npm is generally a given but perhaps it could be worth considering mentioning (somewhere - maybe not here) that it is needed as a baseline for development of the plugin and what versions are recommended etc.

This is a very good point. Along with mentioning the requirement for npm/node, we should point to the `engines` property in our `package.json` to point out the required minimum versions.

Reply:
Ha! Missing dev docs! I've added this to CONTRIBUTING. 

I also added a hint/link to package.json, but that feels overly detailed. Ideally we'd have some high-level guidance here, e.g. "we recommend using node LTS or better".